.github/workflows/build_and_test.yml

```diff
@@ -135,7 +135,7 @@ jobs:
     runs-on: [ self-hosted, gen3, large ]
     container:
       image: 369495373322.dkr.ecr.eu-central-1.amazonaws.com/rust:pinned
-      options: --init
+      options: --init --user root
```

Comment:
Is this required only to install `libcurl4-openssl-dev`?
If so, I'd added this dependency to https://github.com/neondatabase/build/tree/main/rust